data/users/1.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is just test data for now, right? Should not be committed or put into a tests directory, right?

Reply:
I just put it there while I was generating it: we should have more comprehensive data for testing, it's fine to just delete the `data` directory from this PR and set up better test data in a new PR.